docs/readme.md

```diff
@@ -78,11 +78,11 @@ Only one column and content area used. (RIGHT column not used)
 
 - Header
 The custom header `src/gatsby-theme-carbon/components/Header.js` is in this file.
-This is the only one that should be edited. 
+This is the only one that should be edited.
 
-The other files are theme. 
+The other files are theme.
 
-# Links 
+# Links
 [Gatsby Theme: Carbon](https://github.com/carbon-design-system/gatsby-theme-carbon)
 [Gatsby Theme Carbon: Guides](https://gatsby-theme-carbon.now.sh/guides/)
 [Carbon Design](https://www.carbondesignsystem.com/components/link/code/)
```

Comment:
I don't use Eclipse for md files. I use vscode

Reply:
cool, I like vscode. I use it also for node.js and javascript.

Reply:
the trailing whitespaces don't really bother me, but atom edit automatically squashes these.  i mostly just use atom for resolving merge conflincts (because i can't stand those in eclipse).